src/negotiation/taker/negotiator.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { MockBitcoinWallet } from "../../__mocks__/bitcoin_wallet";
-import { Cnd } from "../../cnd/cnd";
+import { Cnd } from "../../__mocks__/cnd";
```

Comment:
This is incorrect according to Jest documentation. The mock should be replacing the class thanks to the jest.mock call.

Reply:
The issue is that you moved the cnd file but you did not move the __mock__ folder that contains the cnd mock.

Reply:
Thanks Franck! should be fixed now :)